### ISSUE TITLE
Anti-factor B antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#11620]
**Preferred term label:**
Anti-factor B antibody positivity

**Synonyms**
anti-FB antibody positivity

**Definition (free text, please give PubMed ID)**
Autoantibodies targeting factor B (a component of the alternative pathway C3 convertase)
PMID: 32034108
PMID: 38252289

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
atypical hemolytic uremic syndrome
C3 glomerulopathy (C3G)
immune-complex membranoproliferative glomerulonephritis (IC-MPGN)
Acute Postinfectious glomerulonephritis

**Your nano-attribution (ORCID)**
0000-0002-3387-1836; 0009-0004-5292-8481

**Comment by pnrobinson** [comment]
@MaudDLVT 
Adding new term: 
- ID: HP:6001537
- label: Anti-factor B antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:23790637|PMID:32034108|PMID:38252289
- def: The presence of an antibody in the blood circulation that is directed against factor B, a component of the alternative C3 convertase.
- comment: Anti-factor B antibodies may be observed in postinfectious glomerulonephritis, C3 glomerulopathy, and immune-complex membranoproliferative glomerulonephritis.
- orcid: orcid:0000-0002-3387-1836
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11620
- synonyms: Anti-FB antibody positivity(NONE)
